haas/cli.py

```diff
@@ -135,10 +156,19 @@ def init_db():
     server.init(init_db=True)
 
 @cmd
-def user_create(username, password):
-    """Create a user <username> with password <password>."""
-    url = object_url('user', username)
-    do_put(url, data={'password': password})
+def user_create(username, password, is_admin):
+    """Create a user <username> with password <password>.
+
+    <is_admin> may be either "admin" or "no-admin", and determines whether
```

Comment:
Could we replace "no-admin" with something like "basic", "regular" or "non-admin"?